lib/chef/resource/windows_ad_join.rb

```diff
@@ -57,6 +57,8 @@ class WindowsAdJoin < Chef::Resource
                default: true
 
       action :join do
+        description "Default. Join the Active Directory domain."
```

Comment:
You can take the default part out. We'll figure that using a bit of logic in the code.